Source/Core/FrustumOutlineGeometry.js

```diff
@@ -0,0 +1,345 @@
+define([
+        './BoundingSphere',
+        './Cartesian3',
+        './Cartesian4',
+        './Check',
+        './ComponentDatatype',
+        './defaultValue',
+        './defined',
+        './Geometry',
+        './GeometryAttribute',
+        './GeometryAttributes',
+        './Matrix3',
+        './Matrix4',
+        './OrthographicFrustum',
+        './PerspectiveFrustum',
+        './PrimitiveType',
+        './Quaternion'
+    ], function(
+        BoundingSphere,
+        Cartesian3,
+        Cartesian4,
+        Check,
+        ComponentDatatype,
+        defaultValue,
+        defined,
+        Geometry,
+        GeometryAttribute,
+        GeometryAttributes,
+        Matrix3,
+        Matrix4,
+        OrthographicFrustum,
+        PerspectiveFrustum,
+        PrimitiveType,
+        Quaternion) {
+    'use strict';
+
+    var PERSPECTIVE = 0;
+    var ORTHOGRAPHIC = 1;
+
+    /**
+     * A description of the outline of a frustum with the given position and orientation.
+     *
+     * @alias FrustumOutlineGeometry
+     * @constructor
+     *
+     * @param {Object} options Object with the following properties:
+     * @param {PerspectiveFrustum|OrthographicFrustum} options.frustum The frustum.
+     * @param {Cartesian3} options.position The position of the frustum.
+     * @param {Quaternion} options.orientation The orientation of the frustum.
+     */
+    function FrustumOutlineGeometry(options) {
```

Comment:
Is it possible for this to share any code with `FrustumGeometry` since all the positions are the same?

Or is that more trouble than it is worth?